applications/pelion_client/README.rst

```diff
@@ -0,0 +1,516 @@
+Also, the device must be provisioned and configured with the credentials for the connection attempt to succeed.
+See `Configuration`_ for details.
+
+The application is built the same way to any other |NCS| application or sample.
+
+.. include:: /includes/build_and_run.txt
+
+.. note::
+    Information about the known issues can be found in |NCS|'s :ref:`release_notes` and on the :ref:`known_issues` page.
+
+Selecting a build type
+======================
+
+Before you start testing the application, you can select one of the build types, depending on your development kit and building method.
+
+Selecting a build type in SES
+-----------------------------
+
+.. include:: /gs_modifying.rst
+   :start-after: build_types_selection_ses_start
+   :end-before: build_types_selection_ses_end
+
+Selecting a build type from command line
+----------------------------------------
+
+.. include:: /gs_modifying.rst
+   :start-after: build_types_selection_cmd_start
+   :end-before: build_types_selection_cmd_end
+
+Testing
+=======
+
+After programming the application and all the prerequisites to your development kit, test the application by performing the following steps:
+
+1. |connect_kit|
+#. Connect to the kit with a terminal emulator to observe log messages (for debug build types).
+#. Reset the development kit.
+#. Turn on the development kit.
+   **LED1** and **LED2** start blinking, which indicates the network is connecting to the Pelion cloud.
+   After several seconds, both LEDs stop blinking and remain turned on.
+   This indicates that the device has established connection to the network backend and the Pelion server.
+#. Log in to Pelion's Device Management Platform.
+#. In the left pane, select :guilabel:`Device directory`.
+   The list of all available devices is displayed.
+
+   .. figure:: /images/pelion_device_directory.png
+      :alt: Pelion's device directory
+
+      Pelion's device directory
+
+   A device that is actively communicating with the server (or was recently communicating with it) is listed with the ``Registered`` state and a green icon.
+   The name of the device shows up on the list (and in the application log) only when it successfully connects to the server.
+#. Select your device from the list of devices.
+   The :guilabel:`Device details` pane appears.
+#. In the :guilabel:`Device details` pane, select the :guilabel:`Resources` tab.
+#. Scroll down to where the value for the Digital Input resource is listed.
+
+   .. figure:: /images/pelion_device_resources.png
+      :alt: Pelion's device directory with the resources tab selected
+
+      Pelion's device directory with the resources tab selected
+
+#. Press Button 1.
+   The Digital Input instance 0 value in Pelion's Device Management Platform increases.
+#. Scroll down to where the value for the Stopwatch resource is listed.
+#. Wait for a couple of seconds to see the value increase.
+
+Dependencies
+************
+
```

Comment:
What do I need to do about it?

Reply:
Absolutely nothing. This is just a standard section that lists the dependencies of the application.